tribuo-classification/enclave/src/main/java/com/r3/conclave/sample/enclave/MlEnclave.java

```diff
@@ -72,16 +70,15 @@ private void trainAndEvaluateModel() {
         LabelEvaluation evaluation = evaluator.evaluate(irisModel, testingDataset);
 
         //send evaluation results back to all the clients using the routingHintToPublicKey mapping
-        for (String key : routingHintToPublicKey.keySet()) {
-            byte[] encryptedReply = postOffice(routingHintToPublicKey.get(key))
+        for (Map.Entry<String, PublicKey> entry : routingHintToPublicKey.entrySet()) {
```

Comment:
You're not using the value from the `entry` object, so the original `keySet()` usage was fine.

Reply:
done